src/api/java/openiss/ws/soap/service/OpenISSSOAPService.java

```diff
@@ -0,0 +1,19 @@
+package api.java.openiss.ws.soap.service;
+
+import javax.jws.WebMethod;  
+import javax.jws.WebService;  
+import javax.jws.soap.SOAPBinding;  
+import javax.jws.soap.SOAPBinding.Style;  
+
+/**
+ * Service Endpoint Interface
+ *
+ * @author zephi
+ */
+
+@WebService  
+@SOAPBinding(style = Style.DOCUMENT)  
```

Comment:
(C) why `Style.DOCUMENT`? Inconsistency. Please two teams align either way.

Reply:
`style` has been changed to `SOAPBinding.Style.RPC`.